modules/openapi-generator/src/main/resources/JavaSpring/libraries/spring-cloud/pom.mustache

```diff
@@ -64,7 +64,18 @@
         <dependency>
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-starter-oauth2</artifactId>
+            <version>2.2.4.RELEASE</version>
         </dependency>
+        <!--Note this is deprecated. resilience4j should be used. To do this, the templates must be adapted.-->
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-starter-netflix-hystrix</artifactId>
+            <version>2.2.7.RELEASE</version>
+        </dependency>
```

Comment:
@MelleD sorry for the delay in following up on this PR. Would it be hard to update the template to remove all reference to hystrix? 

In order words, I would like to remove this dependency as part of this PR since it has been deprecated

(later we can have another PR to add resilience4j support)

Reply:
Yes I think this is a good solution and right way. I will remove hystrix and add to spring boot 2.5.4